addons/mail/static/tests/new/discuss/sidebar_tests.js

```diff
@@ -1112,3 +1113,14 @@ QUnit.test("Can leave channel", async (assert) => {
     });
     assert.containsNone($, ".o-mail-DiscussCategoryItem:contains(General)");
 });
+
+QUnit.test("Channel knowledge is kept after unpin", async (assert) => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["mail.channel"].create({ name: "General", channel_type: "chat" });
+    const { env, openDiscuss } = await start();
+    await openDiscuss(channelId);
+    await click(".o-mail-DiscussCategoryItem-commands [title='Unpin Conversation']");
+    const thread = env.services["mail.store"].threads[createLocalId("mail.channel", channelId)];
+    assert.ok(thread);
+    assert.notOk(thread.is_pinned);
+});
```

Comment:
This test is too technical.
Couldn't you find a better (and more functionally meaningful) test?
I prefer not having this test if you can't find one, because this restricts implementation details for no good reasons.